core/common/src/thrift/file_system_master.thrift

```diff
@@ -271,6 +292,9 @@ service FileSystemMasterClientService extends common.AlluxioService {
     )
     throws (1: exception.AlluxioTException e)
 
+  GetMountTableTResponse getMountTable()
```

Comment:
Put comments here like other APIs. Also can you put this API before `getNewBlockIdForFile` to keep it in order